patroni/api.py

```diff
@@ -468,6 +542,24 @@ def query(self, sql, *params, **kwargs):
         retry = Retry(delay=1, retry_exceptions=PostgresConnectionException)
         return retry(self.server.query, sql, *params)
 
+    def get_postgresql_start_time_epoch(self, retry=False):
+        postgresql = self.server.patroni.postgresql
+        try:
+            if postgresql.state not in ('running', 'restarting', 'starting'):
+                raise RetryFailedError('')
+
+            stmt = ("SELECT extract(epoch from pg_catalog.pg_postmaster_start_time())")
+
+            row = self.query(stmt, retry=retry)[0]
+
+            if row[0]:
+                return row[0]
+            else:
+                return 0
+
+        except (psycopg2.Error, RetryFailedError, PostgresConnectionException):
+            return 0
+
```

Comment:
It could go away, we will use the value returned by `pg_catalog.pg_postmaster_start_time()` as the `datetime` object and format it with strftime(): https://github.com/zalando/patroni/pull/1848/files#diff-13329cd3342d1757c2c9b4dd7a26c0170f4deee5d32eff5c2fd719cc9ff9375bR200
```suggestion
```